src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/Swish/SwishPaymentsResource.cs

```diff
@@ -0,0 +1,28 @@
+﻿using SwedbankPay.Sdk.Payments.SwishPayments;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.Payments
+{
+    public class SwishPaymentsResource : ResourceBase, ISwishResource
+    {
+        public SwishPaymentsResource(HttpClient httpClient) : base(httpClient)
+        {
+        }
+
+        public Task<ISwishPayment> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None)
+        {
+            if (id == null)

```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)